zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1346,6 +1346,42 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
+
+  /incidents/{incidentKey}/resolution:
+    post:
+      tags:
+        - Incident
+      summary: Resolves a given incident. This simply marks the incident as resolved; most likely a call to Update job will be necessary to reset the job’s retries, followed by this call.
```

Comment:
🙃 You can break this summary up into multiple lines like we do in other attributes in this document, I guess. Functionally, this is fine, of course.